lib/streamlit/elements/widgets/button.py

```diff
@@ -517,9 +532,9 @@ def page_link(
             An optional boolean, which disables the page link if set to
             ``True``. The default is ``False``.
         use_container_width : bool
-            An optional boolean, which makes the link stretch its width to
-            match the parent container. The default is ``True`` for page links
-            in the sidebar, and ``False`` for those in the main app.
+            Whether to expand the link's width to fill its parent container.
+            The default is ``True`` for page links in the sidebar and ``False``
+            for those in the main app.
```

Comment:
In testing, I found that this will just break out of its container. I wonder if we should introduce an ellipse of something if we want to prevent line wrap here.

<img width="1649" alt="image" src="https://github.com/streamlit/streamlit/assets/135349133/f8d0812f-5982-4965-b04f-c494347191d5">

Reply:
oh, yes. we should probably add a max width + ellipsis to `page_link` (cc @mayagbarnes)